pkg/status/status.go

```diff
@@ -74,6 +79,7 @@ func (s *Service) LocalSnapshot() (*Snapshot, error) {
 		ConnectedPeers:   connectedPeers,
 		NeighborhoodSize: neighborhoodSize,
 		BatchTotalAmount: s.chainstate.GetChainState().TotalAmount.String(),
+		IsReachable:      s.topologyDriver.IsReachable(),
```

Comment:
instead of getting this field from the peer, we can actually get it directly from our kademlia, 

here we update a peer's rechability https://github.com/ethersphere/bee/blob/master/pkg/topology/kademlia/kademlia.go#L1338

we can add a new function to kademlia that returns of a peer is reachable or not.

Reply:
next we should remove the `topology.Filter{Reachable: true}` filter in status.go so that we try to get the status of ALL of our connected peers.